drivers/hmis/app/graphql/types/hmis_schema/unit.rb

```diff
@@ -68,7 +68,7 @@ def accepting_ce_referrals
       latest_opportunity = load_ar_association(object, :latest_opportunity)
       return false if latest_opportunity.nil?
```

Comment:
this can be removed now, right?

Reply:
I think `accepting_ce_referrals` should still return `false` if there is no opportunity. In other words, even if `active_referral` is nil, the unit may still _not_ be accepting referrals, if it doesn't have an opportunity at all.

Reply:
Oh I see, thanks.

I think we still have the problem of erroneously returning `true` if the unit only has _closed_ opportunities, though. Should this be:
```
return false if latest_opportunity.nil? || latest_opportunity.closed?
```
?

I'm thinking that if the unit only has closed opportunities, active_referral would be nil. But that doesn't mean that it is accepting new ce referrals.

Reply:
Ah yeah absolutely right, thanks for catching that.